java/ql/src/experimental/CWE-094/ScriptEngine.ql

```diff
@@ -0,0 +1,51 @@
+/**
+ * @name Script engine eval
```

Comment:
It's probably best to refer to ScriptEngine here and in the `select` statement for consistency.

```suggestion
 * @name ScriptEngine evaluation
```